scripts/rssFeedGenFull.py

```diff
@@ -0,0 +1,84 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+import PyRSS2Gen
+import re
+
+class NoOutput:
+    def __init__(self):
+        pass
+    def publish(self, handler):
+        pass
+
+class MyRSS2(PyRSS2Gen.RSSItem):
+    def __init__(self, **kwargs):
+        PyRSS2Gen.RSSItem.__init__(self, **kwargs)
+
+    def publish(self, handler):
+        self.do_not_autooutput_description = self.description
+        self.description = NoOutput()
+        PyRSS2Gen.RSSItem.publish(self, handler)
+
+    def publish_extensions(self, handler):
+        handler._out.write('<%s><![CDATA[%s]]></%s>' % ('description', self.do_not_autooutput_description, 'description'))
+
+def extract_releases(html):
+    with open(html, 'r') as f:
+        soup = BeautifulSoup(f.read(), 'lxml')
+        divRelease = soup.find('div', {'id':'release-notes'})
+        releases = divRelease.findAll('div', {'id':re.compile('release-')})
+    return releases
+
+def id_to_rss_date(r):
+    id_str = r.findNext('span').get('id')
+    date_str = id_str.split('-')[1]
+    date_str = date_str + ' 00:00:00'
+    date_obj = datetime.strptime(date_str, '%Y%m%d %H:%M:%S')
+    date_rss = datetime.strftime(date_obj, '%a, %d %b %Y %H:%M:%S %z')
+    return date_rss
+
+def extract_data(r):
+    data = r
+    for elem in data('h2'):
+        elem.extract()
+    return data
+
+def data_to_description(data):
+    data = data.decode('utf-8', 'ignore')
+    description = data.encode('ascii', 'ignore')
+    return description
+
+if __name__ == '__main__':
+    items = []
+    releases = extract_releases('chsdi/static/doc/build/releasenotes/index.html')
```

Comment:
Minor: maybe check if this file exists before trying to open it.

```python
try:
    extract_releases(pathToReleasenotes)
except IOError as e:
    print '%s does not exist' % pathToReleasenotes
    sys.exit(1)
```

It is possible not to have it as the doc is generated.